packages/catalog-model/src/kinds/ApiEntityV1alpha1.ts

```diff
@@ -50,7 +50,13 @@ export class ApiEntityV1alpha1Policy implements EntityPolicy {
     });
   }
 
-  async enforce(envelope: Entity): Promise<Entity> {
+  async enforce(envelope: Entity): Promise<Entity | undefined> {
+    if (
+      KIND !== envelope.kind ||
+      !API_VERSION.includes(envelope.apiVersion as any)
+    ) {
+      return undefined;
```

Comment:
it's a bit annoying that you can't invert the condition and skip the `return undefined` - the linter complains that all return paths do not return a value, which is very helpful in 99% of cases but annoying when you really wanted that behaviour :)

Reply:
Could actually be worth having some internal helper for `match(envelope, KIND, API_VERSION)`. Would make a ternary look not terrible too